src/handlers/notification/index.js

```diff
@@ -131,16 +131,19 @@ const consumeMessage = async (error, message) => {
 
 const processMessage = async (msg) => {
   try {
-    Logger.info('Notification::processMessage')
+    const { metadata, from, to, content, id } = msg.value
+
+    Logger.info(`[cid=${id}, fsp=${from}, source=${from}, dest=${to}] ~ Notification::processMessage`)
     if (!msg.value || !msg.value.content || !msg.value.content.headers || !msg.value.content.payload) {
       throw new Error('Invalid message received from kafka')
     }
-    const { metadata, from, to, content, id } = msg.value
+
     const { action, state } = metadata.event
```

Comment:
Should this not be msg.value.metadata.event

Reply:
Nope, because we have defined the "metadata" object in line 138.

Do you not agree?